Marlin/Configuration.h

```diff
@@ -400,10 +418,10 @@ const bool Z_MAX_ENDSTOP_INVERTING = false; // set to true to invert the logic o
 #define BED_CENTER_AT_0_0  // If defined, the center of the bed is at (X=0, Y=0)
 
 //Manual homing switch locations:
-// For deltabots this means top and center of the Cartesian print volume.
+// For deltabots this means top and center of the cartesian print volume.
 #define MANUAL_X_HOME_POS 0
 #define MANUAL_Y_HOME_POS 0
-#define MANUAL_Z_HOME_POS 300  // For delta: Distance between nozzle and print surface after homing.
+#define MANUAL_Z_HOME_POS 280.67 //300  // For delta: Distance between nozzle and print surface after homing.
```

Comment:
Two reasons - it's a better fit for the KosselPro as well as a closer approximation for RichCattell's autocal routines to start from.  The process will set the height such that any endstop adjustment needed will be negative and will tweak further during the process such that the bed height tends to 0.00.

After the process you can fine-tune M666 Hn.nn to set the head clearance correctly.